src/manifest.js

```diff
@@ -114,6 +146,20 @@ export function getManifest(config, visitedManifestUrls = []) {
 	return promise;
 }
 
+export function getAllManifests(config) {
+	return new Promise((resolve, reject) => {
+		getManifest(config)
+			.then((manifest) => {
+				resolve({
+					flat: manifest,
```

Comment:
perhaps `flat: { ...manifest}` so that middlewares can't actually mutate the manifest? Or are you thinking that we would want them to be able to?